examples/upload/src/controller.erl

```diff
@@ -0,0 +1,186 @@
+-module(controller).
+
+-export([
+    execute/2,
+	process/4,
+	resource_exists/1,
+	content_types_provided/1,
+	content_types_accepted/1,
+	allowed_methods/1,
+	post_is_create/1,
+	create_path/1
+]).
+
+execute(Req, Env) ->
+	{ok, Req, Env#{ cowmachine_controller => ?MODULE }}.
+
+process(<<"GET">>, _ContentType, _Accepted, Context) ->
+	{true, Context};
+process(<<"POST">>, _ContentType, _Accepted, Context) ->
+	case cowmachine_req:resp_content_type(Context) of

```

Comment:
I think we need to get the content-type of the payload here, which is the AcceptedCT.
This is the same as the `AcceptedCT` in the `process/4` parameters.

Reply:
Thank you @mworrell for describing this. I have added this simplification.